### PR TITLE
logging to help debug tests

### LINKED ISSUE #82: investigate flaky tests
It looks like the tests introduced w/ the debugger PR are flaky on the bots:

https://travis-ci.org/flutter/devtools/jobs/474983704

We should investigate the timing issues and ensure the tests are reliable.

### PR DESCRIPTION
**Pull request by devoncarew**
Adding logging and delays to help de-flake tests.

fix #82

### CODE REVIEW

File: test/integration_tests/integration.dart [hunk 1]
```diff
@@ -88,12 +95,18 @@ class BrowserManager {
...
     final ChromeTab tab =
         await chromeProcess.connectToTabId('localhost', targetId);
-    await tab.connect();
+    await tab.connect(verbose: true);
+
+    await delay();
 
     await tab.wipConnection.target.activateTarget(targetId);
 
+    await delay();
```
Comment (devoncarew) [inline test/integration_tests/integration.dart]: This turns out to be the important delay; I think chrome wasn't quite ready for the url to change here, and for it to load new page contents.

**Comment by devoncarew** [comment]
@jacob314, @kenzieschmoll, this is ready for review now

- several delays added, including at least one that seems important (a small delay after creating a new browser tab, and before we ask it to load content
- lots more logging added, to help us debug any issues encountered in the future

The bots seem reliable now after this PR.

File: test/integration_tests/integration.dart (hunk 1 above)
Comment (devoncarew) [reply test/integration_tests/integration.dart]: 👍, I'll create an issue to track this.

File: web/index.html [hunk 2]
```diff
@@ -22,13 +22,11 @@
     <link rel="stylesheet" href="packages/devtools/debugger/debugger.css">
     <link rel="stylesheet" href="packages/devtools/logging/logging.css">
 
-    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```
Comment (jacob314) [inline web/index.html]: We weren't actually using the charts .js library, correct?

Comment (devoncarew) [reply web/index.html]: Yeah, this was from a while ago when charts were displayed via google charts. We don't anymore - we just use straight svg.

**Pull request merged**